packages/design-system/src/connect/components/home-screen-item/home-screen-item.tsx

```diff
@@ -0,0 +1,46 @@
+import PeopleFill from "@/assets/icon-components/PeopleFill";
+import { ComponentPropsWithRef, forwardRef } from "react";
+import { twMerge } from "tailwind-merge";
+
+type InputProps = ComponentPropsWithRef<"input">;
+type HomeScreenItemProps = Omit<InputProps, "className"> & {
+  readonly icon?: React.JSX.Element;
+  readonly title: string;
+  readonly description?: string;
+  readonly containerClassName?: string;
+  readonly sharable?: boolean;
```

Comment:
"shareable" typo

Reply:
Shareable is the only prop directly related to a Drive.
I'd keep this one simple and create a separate HomeScreenDriveItem component.
This can receive the drive location (public, local, ...) and set the correct icon.